create-a-commandl-line-program/includes/1-introduction.md

```diff
@@ -0,0 +1,16 @@
+# Introduction
+
+In this final module, we'll develop a modern, real-world Rust program.
+
+Our application will be a command-line to-do tracker that records our
+tasks into a text file, display them as a list in the terminal and
+lets us mark them complete.
+
```

Comment:
Any helpful learning objectives you can think of?

Reply:
What about something like:

    Learn how to develop a real-world command-line program using battle-tested third party crates for command line parsing and error handling 

Or
    Learn how to assemble a real-world program using dedicated modules and crates from the Rust community

Reply:
I like the first option! I'll add it in now.